src/NewTwitchApi/Resources/UsersApi.php

```diff
@@ -12,43 +12,37 @@ class UsersApi extends AbstractResource
     /**
      * @throws GuzzleException
      */
-    public function getUserByAccessToken(string $accessToken, bool $includeEmail = false): ResponseInterface
+    public function getUserByAccessToken(string $accessToken): ResponseInterface
```

Comment:
Why did you remove `$includeEmail` from the "get user" calls? This is a valid optional parameter. If the caller wants user's email addresses, the call needs the have the proper scope sent. If not, the scope doesn't need to be sent. I wanted to leave this option open for callers, since Twitch makes it available in the API.

I see in the PR description, you said:

```
 Scopes are implicitly bound to access tokens, thus it is not required that they be included in individual request params.
```

Correct. The scope isn't required, but is an option that can be added to the get users API call. That's how I understand the documentation, anyway.

https://dev.twitch.tv/docs/api/reference/#get-users

Reply:
If you're access token has the appropriate scope, then the email address will be returned in the payload. If the access token does not have the appropriate scope, the email address will be omitted.

If you're querying for more than one users, some users may or may not have a email address in the returned payload.

Adding the scope as a query param will do nothing unless you have a correctly scoped access token. It is redundant to include it in the request.

Reply:
Hm. Ok, I'll go with this and approve the removal. I'm not currently using that parameter, anyway. I just had it there to match what the API allows you to request. If this needs to be added back in the figure, it always can be. :-)